draft-fossati-tls-exported-attestation.md

```diff
@@ -289,10 +317,32 @@ session has already completed remote attestation before the session can be used
 
 ## Evidence Freshness
 
-The Evidence carried in cmw_attestation does not require an additional freshness mechanism, such as a nonce or timestamp, since freshness is inherently provided by the certificate_request_context in the authenticator request.
+The Evidence carried in cmw_attestation does not require an additional freshness mechanism, such as a nonce {{RA-TLS}} or timestamp, since freshness is inherently provided by the certificate_request_context in the authenticator request.
 
 The evidence presented in this protocol is valid only at the time it is generated and presented. To ensure that the attested peer remains in a secure state, remote attestation may be re-initiated periodically. In the current protocol, this can be achieved by initiating a new Exported Authenticator-based post-handshake authentication exchange, which will generate a new certificate_request_context to maintain freshness.
 
+# Privacy Considerations
+
+In this section, we are assuming that the Attester is a TLS client, representing an individual person.
+We are concerned about the potential leakage of privacy-sensitive information about that person, such as the correlation of different connections initiated by them.
+
+In background-check model, the Verifier not only has access to detailed information about the Attester's TCB through Evidence, but it also knows the exact time and the party (i.e., the RP) with whom the secure channel establishment is attempted [RA-TLS].
```

Comment:
Is that `[RA-TLS]` at the end correct syntax? I'm probably still clueless about how this format works...

Reply:
It should be `{{RA-TLS}}`